src/qml/qgismobileapp.qml

```diff
@@ -3589,6 +3593,11 @@ ApplicationWindow {
         displayToast(qsTr('Signed in'))
         // Go ahead and upload pending attachments in the background
         platformUtilities.uploadPendingAttachments(cloudConnection);
+
+        var cloudProjectId = QFieldCloudUtils.getProjectId(qgisProject.fileName)
+        if (cloudProjectId !== '') {
```

Comment:
```suggestion
        if (cloudProjectId) {
```

Just to be slightly safer here, it might return null or undefined in the future.